<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment>
  * Released as open source by NCC Group Plc - http://www.nccgroup.com/
  *
  * Developed by Gabriel Caudrelier, gabriel dot caudrelier at nccgroup dot com
  *
  * https://github.com/nccgroup/pip3line
  *
  * Released under AGPL see LICENSE for more information
</comment>
 <class>CutWidget</class>
 <widget class="QWidget" name="CutWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="labelAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="fromLabel">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="fromSpinBox">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lengthLabel">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="lengthSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="everythingCheckBox">
     <property name="text">
      <string>Everything else (infinite length)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
